examples_tests/03.GPU_Mesh/main.cpp

```diff
@@ -4,15 +4,35 @@
 #include <irrlicht.h>
 
 //! I advise to check out this file, its a basic input handler
-#include "../common/QToQuitEventReceiver.h"
-
-//#include "../../ext/ScreenShot/ScreenShot.h"
-
+#include "../../ext/ScreenShot/ScreenShot.h"
+#include "../../src/irr/builtin/builtinResources.h"
```

Comment:
don't touch my samples, try to embed one of the shaders in `IAssetManager` instead of having it as a rawstring and test (example 35 uses builtin shaders)